app/views/distributions/show.html.erb

```diff
@@ -54,6 +55,7 @@
       </div>
     </div>
     <div class="box-footer with-border">
+      <%= restore_button_to picked_up_distribution_path(@distribution), {text: "Complete pickup", size: "lg"} if @distribution.scheduled? %>
```

Comment:
I appreciate you using a helper that is using the correct REST action! :) 

I think the context here is a little confusing though, since you aren't actually restoring anything. Please feel free to create a new action in `app/helpers/ui_helper.rb` that is more appropriate for this. Perhaps something like:
```ruby
def update_button_to(link, options = {})
 # ...
end
```
You can probably copy some of the code from `restore_button_to`. Let me know if you have any questions or find any of this confusing. :+1: